packages/ra-ui-materialui/src/input/SelectArrayInput.tsx

```diff
@@ -11,13 +11,15 @@ import {
     FormHelperText,
     FormControl,
     Chip,
+    OutlinedInput,
 } from '@mui/material';
 import {
     ChoicesProps,
     FieldTitle,
     useInput,
     useChoicesContext,
     useChoices,
+    useTranslateLabel,
```

Comment:
Almost there but you forgot to remove this import too.

I need to request this change because otherwise our CI will raise a warning here that we'll need to fix later, so we might as well do it now.